libs/langchain/langchain/memory/chat_memory.py

```diff
@@ -26,9 +26,12 @@ def _get_input_output(
         else:
             prompt_input_key = self.input_key
         if self.output_key is None:
-            if len(outputs) != 1:
-                raise ValueError(f"One output key expected, got {outputs.keys()}")
-            output_key = list(outputs.keys())[0]
+            if len(outputs) == 1:
+                output_key = list(outputs.keys())[0]
+            elif "output" in outputs:
+                output_key = "output"
```

Comment:
can we add some logging here? I don't think this is expected, and id prefer if people set it manually. so im fine adding this, but id want good logging so they know whats going on

Reply:
how about adding a warning message in line 32:
`warnings.warn(f"'{self.__class__.__name__}' got multiple output keys: {outputs.keys()}. The default 'output' key is being used. If this is not desired, please manually set 'output_key'.")`